packages/eslint-plugin-office-addins/src/utils/utils.ts

```diff
@@ -31,7 +31,14 @@ export function findTopLevelExpression(
 }
 
 export type OfficeApiReference = {
-  operation: "Get" | "Load" | "Sync" | "Read";
+  /**
+   * Get: An OfficeJs object, which is created when calling a `get` type function
+   * Load: A reference to `object.load()` type call
+   * Method: The reference is calling a method. Ex: `object.methodCall()`
```

Comment:
So we are labeling 'methodCall' as method and not 'object' correct?  This sounds like we're labeling 'object' as calling a method.

Reply:
We are labeling the object as calling a method right now.

Reply:
I don't think that is correct.  We are examining in references and categorizing them, but this is suggesting not categorizing the reference itself, it's categorizing the parent of the reference.  We want to indicate that a reference is a method (not that it's parent is calling a method) and then act on that reference later when looking for things that need load and sync calls.

Reply:
I dont know for sure how we could do this the way you are suggesting. The references, for example, in the code below:
```
var range = worksheet.getSelectedRange();
range.getCell(0,0);
```
are only `range` (var range) and `range`(range.getCell). So we don't have a way to categorize `getCell`. We need to categorize the `range` in the `range.getCell` call and that is what is being done.

Reply:
Why can't we categorize 'getCell'?  Isn't it a reference in our list of references and aren't we are currently categorizing it as a property without a load and sync (thus it is getting falsely flagged by the existing rules)?

Reply:
It is not considered a reference, thus it is not categorized.

Reply:
Then how is it getting picked up by "scope.references.forEach((reference) => ..." to test in the first place?  The method is examining every reference in the scope (including things like 'let 1 = 1; let x = 1' which have nothing to do with office.js) and deciding if it is something we should pay attention to later (adding it to the apiReferences collection) or not.  When adding it to the collection we further classify.

Reply:
I'm saying the reference picked up by the function is `range` and not the `getCell` call. So we can only categorize `range`, we don't get a reference for the method call name, only for the object that called it.